wso2-synapse-unit-test-plugin/src/main/java/org/wso2/synapse/unittest/UnitTestCasesMojo.java

```diff
@@ -414,6 +415,16 @@ private void stopTestingServer() {
             } catch (Exception e) {
                 getLog().error("Error in closing the server", e);
             }
+        } else if (server.getServerType() != null && server.getServerType().equals(REMOTE_SERVER)) {
+            try {
+                if (getLog().isDebugEnabled()) {
+                    getLog().debug("Stopping docker container ");
+                }
+                Runtime.getRuntime().exec("docker stop wso2-mi-container").waitFor();
```

Comment:
Is the container name always fixed? We might have to rethink whether we are supporting only docker containers under REMOTE_SERVER.

Reply:
Undo these changes.